static/absolute/js/searchPage.js

```diff
@@ -24,14 +27,35 @@ window.onload = function () {
         return;
       }
     } else {
-      // TODO(ved): Use API to convert address --> long/lat
-      alert("This is not yet supported");
-      return;
+      if (/[`!#$%\^&*+=\-\[\]\\';/{}|\\":<>\?]/.test(address)) {
```

Comment:
be careful with assuming what characters are invalid/valid in queries like this. anything to do with localization and globalization is likely more complicated than you would expect. does filtering these characters add value, as compared to simply making the query and returning the results? are you absolutely sure it is impossible to construct a valid address with any of these characters?

Reply:
I changed the input to include all characters.

There was no reason to take out the special characters aside from them not being common in addresses and them needing to be encoded in the geocode URLs.